doc/src/manual/environment-variables.md

```diff
@@ -0,0 +1,313 @@
+# Environment Variables
+
+The environment variables that Julia uses generally start with `JULIA`. If
```

Comment:
Should we add here in the beginning how to actually set/modify the variables? And perhaps mention that it could be done conveniently in the `.juliarc.jl` file?